apps/ai-hero/src/lib/lists-query.ts

```
@@ -416,3 +418,54 @@ export async function deleteList(id: string) {
+	if (!ability.can('update', 'Content')) {
+		throw new Error('Unauthorized')
+	}
+	const item = await db.query.contentResource.findFirst({
+		where: eq(contentResource.id, itemId),
+	})
+
+	if (!item) throw new Error('item not found')
+	let result
+	switch (item.type) {
+		case 'post':
+			const parsedPost = PostSchema.parse(item)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)